### PR TITLE
Cleanup linter errors.

### PR DESCRIPTION
**Pull request by devinleighsmith**
Fix missing packages.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -2,6 +2,9 @@
 * text=auto
 
 # Declare files that will always have LF line endings on checkout.
+*.tsx text eol=lf
```
Comment (Fosol) [inline .gitattributes:5]: Thank you for this!

File: frontend/package.json [hunk 2]
```diff
@@ -4,16 +4,18 @@
   "private": true,
   "dependencies": {
     "@bcgov/bc-sans": "1.0.1",
-    "@react-keycloak/web": "1.0.0-alpha.6",
+    "@react-keycloak/web": "2.1.0",
```
Comment (Fosol) [inline frontend/package.json:7]: Wow happy to get off alpha!

File: frontend/src/actionCreators/parcelsActionCreator.tsx [hunk 3]
```diff
@@ -14,8 +14,8 @@ export const fetchParcels = (parcelBounds: API.IParcelListParams | null) => (
 ) => {
   if (
     !parcelBounds ||
-    (parcelBounds?.neLatitude != parcelBounds?.swLatitude &&
-      parcelBounds.neLongitude != parcelBounds.swLongitude)
+    (parcelBounds?.neLatitude !== parcelBounds?.swLatitude &&
+      parcelBounds.neLongitude !== parcelBounds.swLongitude)
```
Comment (Fosol) [inline frontend/src/actionCreators/parcelsActionCreator.tsx:18]: Shouldn't these all be `parcelBounds?`?

Comment (devinleighsmith) [reply frontend/src/actionCreators/parcelsActionCreator.tsx:18]: line 16 actually makes the ? redundant.

**Pull request merged**